src/gapp_login/wechat/endpoints.py

```diff
@@ -83,6 +96,42 @@ async def login_wechat(
     return rv
 
 
+@router.post("/users/update/wxa")
+async def update_wechat_account_info(
+    user: User = Security(require_user),
+    encrypted_data: str = Form(...),
+    iv: str = Form(...),
+    app_id: str = Form(...),
+):
+    encrypted_data = base64.b64decode(encrypted_data)
+    iv = base64.b64decode(iv)
+    identity = await (
+        WeChatIdentity.query.select_from(User.outerjoin(WeChatIdentity))
+        .where(User.id == user.id)
+        .where(WeChatIdentity.wechat_app_id == app_id)
+        .where(db.func.starts_with(WeChatIdentity.idp, "WECHAT"))
```

Comment:
@fantix Is there any way to get correct identity without pass the app_id paramter.

Reply:
I think we could add `client_id` to the authorization code. I'll add to authlib-gino with a PR.

Reply:
Okay. Then I will update this PR after that.

Reply:
fixed